pkg/addon/controllers/addonconfiguration/graph.go

```diff
@@ -341,3 +383,11 @@ func desiredConfigsEqual(a, b addonConfigMap) bool {
 
 	return true
 }
+
+func initClusters(pdTracker *clusterv1beta1.PlacementDecisionClustersTracker) (sets.Set[string], error) {
```

Comment:
I am not quite sure why we need this func

Reply:
The input pdTracker does not include any clusters, so want to call Get() to fill in the existingScheduledClusterGroups and return the existing clusters. Might need to enhance the PlacementDecisionClustersTracker functions rather than add initClusters() here.

Reply:
remove this func